lib/install/config/loaders/installers/elm.js

```diff
@@ -1,5 +1,9 @@
+const path = require('path')
+
+const elmSource = path.join(__dirname, '..', '..', '..')
```

Comment:
I'm afraid that won't work because `elmSource` must point to the root of the generated Rails app where the `elm-package.json` is located. This allows Webpack to load the configuration as defined in the `elm-package.json` along with any Elm package dependencies.